services/file.py

```diff
@@ -16,6 +16,7 @@ async def get_document_from_file(file: UploadFile) -> Document:
     print(f"extracted_text:")
     # get metadata
     metadata = DocumentMetadata()
+    metadata.source = "file"
```

Comment:
When setting source, using Source.file from model.models is required vs the string

Reply:
Of course. Hardcoding was a mistake - fixed that and tested. 
Also: I don't think it is mandatory to use the Source.type in the filter during query. It was tested and works without that.